crates/dojo-defi/src/constant_product_market/components.cairo

```diff
@@ -123,7 +124,8 @@ impl MarketImpl of MarketTrait {
         let reserve_quantity = Fixed::new_unscaled(reserve_quantity, false);
 
         // Normalize quantity
-        let quantity = quantity.into().try_into().unwrap() * SCALING_FACTOR;
+        let quantity: felt252 = quantity.into();
+        let quantity: u128 = quantity.try_into().unwrap() * SCALING_FACTOR;
```

Comment:
I don't think we need intermediate `felt252` conversions. We can safely go from `usize -> u128` directly here and in other places

```
let quantity: u128 = quantity.into() * SCALING_FACTOR;
```

Reply:
Thanks! Will add it in #283